package.json

```diff
@@ -15,26 +15,39 @@
   "publishConfig": {
     "access": "public"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged && npm run lint"
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/teleporthq/teleport-markdown-uidl-generator.git"
+  },
   "scripts": {
     "test": "jest",
     "test:watch": "jest --watch",
     "clean": "rimraf dist",
```

Comment:
We are still using `rimraf` here. But removed from `devDependencies` 🤔

Reply:
it's in devDependencies